manager-bundle/src/HttpKernel/JwtManager.php

```diff
@@ -130,7 +130,7 @@ public function parseCookie(string $data): ?array
     {
         $token = $this->parser->parse($data);
 
-        if ($token->isExpired() || !$token->verify($this->signer, $this->secret)) {
+        if ($token->isExpired(new \DateTimeImmutable()) || !$token->verify($this->signer, new Key($this->secret))) {
```

Comment:
I think these are the changes (along with increasing the requirement for `lcobucci/jwt` from `^3.2` to `^3.4`) that have to be backported to **4.9**, in order to fix https://github.com/contao/contao-manager/issues/612 on hosting environments where `E_USER_DEPRECATED` is not excluded from the displayed errors on the CLI, correct? /cc @aschempp

Reply:
Yes, that should fix the issue.